src/plugins/intel_gpu/src/kernel_selector/kernels/resample/resample_kernel_onnx.cpp

```diff
@@ -0,0 +1,173 @@
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableInputDataType(Datatype::UINT8);
+    k.EnableInputDataType(Datatype::INT8);
+    k.EnableOutputDataType(Datatype::F16);
+    k.EnableOutputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::UINT8);
+    k.EnableOutputDataType(Datatype::INT8);
+
+    // 5d formats
+    k.EnableInputLayout(DataLayout::b_fs_zyx_fsv16);
+    k.EnableOutputLayout(DataLayout::b_fs_zyx_fsv16);
+    k.EnableInputLayout(DataLayout::bs_fs_zyx_bsv16_fsv16);
+    k.EnableOutputLayout(DataLayout::bs_fs_zyx_bsv16_fsv16);
+    k.EnableInputLayout(DataLayout::bs_fs_zyx_bsv32_fsv16);
+    k.EnableOutputLayout(DataLayout::bs_fs_zyx_bsv32_fsv16);
+    k.EnableInputLayout(DataLayout::b_fs_zyx_fsv32);
+    k.EnableOutputLayout(DataLayout::b_fs_zyx_fsv32);
+    k.EnableInputLayout(DataLayout::bs_fs_zyx_bsv16_fsv32);
+    k.EnableOutputLayout(DataLayout::bs_fs_zyx_bsv16_fsv32);
+    k.EnableInputLayout(DataLayout::bs_fs_zyx_bsv32_fsv32);
+    k.EnableOutputLayout(DataLayout::bs_fs_zyx_bsv32_fsv32);
+
+    k.EnableDifferentTypes();
+    k.EnableTensorOffset();
+    k.EnableTensorPitches();
+    k.EnableBatching();
+    k.EnableReampleType(ResampleType::LINEAR_ONNX);
+    k.EnableSubGroup();
+    k.EnableSubGroupShort();
+    return k;
+}
+
+ResampleKernelBase::DispatchData ResampleKernelOnnx::SetDefault(const kernel_selector::resample_params &arg) const {
+    DispatchData dispatchData;
+    std::vector<std::vector<Tensor::DataChannelName>> dims_by_gws;
+
+    const auto& out = arg.outputs[0];
+
+    auto opt_x_block_size = GetOptimalBlockSize(arg);
+    if (out.X().v > 32 && opt_x_block_size == 1) {
+        opt_x_block_size = GetOptimalDivisor(out.X().v, 32);
+    }
+
+    dispatchData.gws[0] = CeilDiv(out.X().v, opt_x_block_size) * out.Y().v * out.Z().v;
+    dispatchData.gws[1] = Align(out.Feature().v, sub_group_size);
+    dispatchData.gws[2] = arg.outputs[0].Batch().v;
+
+    dispatchData.lws[0] = 1;
+    dispatchData.lws[1] = sub_group_size;
+    dispatchData.lws[2] = 1;
+
+    if (arg.outputs[0].GetLayout() == DataLayout::bs_fs_zyx_bsv32_fsv16 ||
+        arg.outputs[0].GetLayout() == DataLayout::bs_fs_zyx_bsv32_fsv32) {
+        dispatchData.lws[2] = GetOptimalDivisor(dispatchData.gws[2]);
+    }
+
+    return dispatchData;
+}
+
+KernelsPriority ResampleKernelOnnx::GetKernelsPriority(const Params& /*params*/, const optional_params& /*options*/) const {
+    return FORCE_PRIORITY_4;
+}
+
+bool ResampleKernelOnnx::Validate(const Params& p, const optional_params& o) const {
+    const resample_params& params = static_cast<const resample_params&>(p);
+    if (params.resampleType != ResampleType::LINEAR_ONNX)
+        return false;
+
+    if (!Parent::Validate(p, o))
+        return false;
+
+    if (p.GetType() != KernelType::RESAMPLE || o.GetType() != KernelType::RESAMPLE)
+        return false;
+
+    if (params.inputs.empty())
```

Comment:
These 2 checks above are a part of Parent::Validate(), so they're not needed here.

Reply:
It was copy/pasted from ResampleKernelOpt. Ok, delete in both of them.